documentation/package-table.md

```diff
@@ -25,236 +25,236 @@ Reference notes:
-[linux-arm64-targz-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-linux-arm64.tar.gz.sha
+[linux-arm64-targz-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-linux-arm64.tar.gz.sha512
 
 [linux-arm64-badge-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/linux_arm64_Release_version_badge.svg?no-cache
 [linux-arm64-version-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/productCommit-linux-arm64.txt
 [linux-arm64-targz-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-linux-arm64.tar.gz
-[linux-arm64-targz-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-linux-arm64.tar.gz.sha
+[linux-arm64-targz-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-linux-arm64.tar.gz.sha512
 
 [linux-arm64-badge-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/linux_arm64_Release_version_badge.svg?no-cache
 [linux-arm64-version-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/productCommit-linux-arm64.txt
 [linux-arm64-targz-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-arm64.tar.gz
-[linux-arm64-targz-checksum-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-arm64.tar.gz.sha
+[linux-arm64-targz-checksum-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-arm64.tar.gz.sha512
 
 [rhel-6-badge-main]: https://aka.ms/dotnet/10.0.1xx/daily/rhel.6_x64_Release_version_badge.svg?no-cache
 [rhel-6-version-main]: https://aka.ms/dotnet/10.0.1xx/daily/productCommit-rhel.6-x64.txt
 [rhel-6-targz-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-rhel.6-x64.tar.gz
-[rhel-6-targz-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-rhel.6-x64.tar.gz.sha
+[rhel-6-targz-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-rhel.6-x64.tar.gz.sha512
 
 [rhel-6-badge-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/rhel.6_x64_Release_version_badge.svg?no-cache
 [rhel-6-version-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/productCommit-rhel.6-x64.txt
 [rhel-6-targz-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-rhel.6-x64.tar.gz
-[rhel-6-targz-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-rhel.6-x64.tar.gz.sha
+[rhel-6-targz-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-rhel.6-x64.tar.gz.sha512
 
 [rhel-6-badge-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/rhel.6_x64_Release_version_badge.svg?no-cache
 [rhel-6-version-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/productCommit-rhel.6-x64.txt
 [rhel-6-targz-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-rhel.6-x64.tar.gz
-[rhel-6-targz-checksum-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-rhel.6-x64.tar.gz.sha
+[rhel-6-targz-checksum-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-rhel.6-x64.tar.gz.sha512
```

Comment:
Do we still have these files? If so, we should really rename them to their more applicable base targets (as we don't support RHEL6 any more and haven't for years)

Reply:
I want to keep this scoped to the PR's intent. This table will be replaced by VMR's table eventually.